integration-tests/deploy_R_models.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import absolute_import, print_function
```

Comment:
Thanks @Corey-Zumar for the review.
Most of the tests in the `/integration-tests`  like `deploy_PySpark_models.py` , `many_apps_many_models.py`  
etc are not Python unit tests. Any particular reason to make it different from the rest of model related tests?